File: app/src/androidTest/main/java/edu/cascadia/emilio/foncfphoto_points/accountForm.java
package edu.cascadia.emilio.foncfphoto_points;

import android.graphics.Color;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.EditText;
import android.widget.Toolbar;

public class accountForm extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_account_form);
        Toolbar myToolbar = (Toolbar) findViewById(R.id.my_toolbar);

        // Get the widget reference for XML layout
        EditText first = (EditText) findViewById(R.id.ac_first);
        EditText last = (EditText) findViewById(R.id.ac_last);
        EditText email = (EditText) findViewById(R.id.ac_email);
        EditText pin = (EditText) findViewById(R.id.ac_pin);

        first.setHintTextColor(Color.WHITE);
        last.setHintTextColor(Color.WHITE);
        email.setHintTextColor(Color.WHITE);
        pin.setHintTextColor(Color.WHITE);
    }
}

File: app/src/androidTest/main/java/edu/cascadia/emilio/foncfphoto_points/accountForm.kt
package edu.cascadia.emilio.foncfphoto_points

import android.graphics.Color
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.EditText
import android.widget.Toolbar

class accountForm : AppCompatActivity() {

    @Override
    protected fun onCreate(savedInstanceState: Bundle) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_account_form)
        val myToolbar = findViewById(R.id.my_toolbar) as Toolbar

        // Get the widget reference for XML layout
        val first = findViewById(R.id.ac_first) as EditText
        val last = findViewById(R.id.ac_last) as EditText
        val email = findViewById(R.id.ac_email) as EditText
        val pin = findViewById(R.id.ac_pin) as EditText

        first.setHintTextColor(Color.WHITE)
        last.setHintTextColor(Color.WHITE)
        email.setHintTextColor(Color.WHITE)
        pin.setHintTextColor(Color.WHITE)
    }
}
